homeassistant/components/alexa/entities.py

```diff
@@ -138,7 +138,9 @@ def friendly_name(self):
 
     def description(self):
         """Return the Alexa API description."""
-        return self.entity_conf.get(CONF_DESCRIPTION, self.entity.entity_id)
+        return self.entity_conf.get(CONF_DESCRIPTION) or "{} via Home Assistant".format(
```

Comment:
Please use f-strings instead of str.format.

Reply:
Maybe entity domain instead friendly name?

Reply:
Let's use `entity_id`, friendly name is already set as name. This is really to help users find their way back which entity it is in HA.